user/languages/matlab.md

```diff
@@ -24,16 +24,15 @@ language: matlab
 
 </aside>
 
-{: .warning}
-> MATLAB builds are not currently supported. MATLAB releases newer than R2024a are not available for use.
-
 This guide covers build environment and configuration topics specific to
 [MATLAB&reg;](https://www.mathworks.com/products/matlab.html) and [Simulink&reg;](https://www.mathworks.com/products/simulink.html) projects. Please make sure to read our
 [Tutorial](/user/tutorial/) and
 [general build configuration](/user/customizing-the-build/) guides first.
 
 {: .warning}
-> Currently, MATLAB builds are available only for public projects in Linux&reg; environments.
+> Builds using the MATLAB language are no longer supported. MATLAB releases newer than R2024a are not available for use.
+>
+> MATLAB builds are available only for public projects in Linux&reg; environments.
```

Comment:
It's not a good idea to have back to back notes. If you want to keep this, it should be a regular sentence instead of a note. Or you can combine everything into a single note.